src/Resources/contao/languages/en/tl_content.php

```diff
@@ -13,9 +13,45 @@
  */
 
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_legend'] = 'Column configuration';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_large'][0] = 'Desktop XL columns';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_large'][1] = 'Example: 2, 3, 1-2-1, 1-3. Max. 12 cols. Explanation: The desired number of evenly distributed columns (e.g. 3) or a statement following the syntax 1-2-1 for 3 columns with a doubled middle column. Note: Per column you\'ll need one element. Single elements aren\'t split automatically.';
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_large'][0] = 'Desktop columns';
-$GLOBALS['TL_LANG']['tl_content']['rs_columns_large'][1] = 'Example: 2, 3, 1-2-1, 1-3. Max. 6 cols. Explanation: The desired number of evenly distributed columns (e.g. 3) or a statement following the syntax 1-2-1 for 3 columns with a doubled middle column. Note: Per column you\'ll need one element. Single elements aren\'t split automatically.';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_large'][1] = 'Example: 2, 3, 1-2-1, 1-3. Max. 12 cols. (optional)';
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_medium'][0] = 'Tablet columns';
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_medium'][1] = 'Example: 2, instead of 1-1-2 on desktop... number and distribution of the columns for visitors with smaller resolution (optional).';
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_small'][0] = 'Mobile columns';
 $GLOBALS['TL_LANG']['tl_content']['rs_columns_small'][1] = 'Usually 1 or 2... the distribution for even smaller screens (optional, but recommended).';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_xsmall'][0] = 'Mobile XS columns';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_xsmall'][1] = 'Usually 1 or 2... the distribution for even smaller screens (optional, but recommended).';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutter'][0] = 'Columns gutter';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutter'][1] = 'Size of the gutter between the columns.';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['none'] = 'None';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['s'] = 'Small';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['m'] = 'Medium';
+$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['l'] = 'Large';
```

Comment:
```suggestion
$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['l'] = 'Large';
$GLOBALS['TL_LANG']['tl_content']['rs_columns_gutters']['h'] = 'Huge';
```
Was just added in the most recent commit on the v3 branch.